tests/unit/rule_strings/implemented_rule.txt

```diff
@@ -0,0 +1,14 @@
+
+rule:
+    name: "foo"
+    input:
+        implementation_inputs=['foo', 'bar'],
+        validation="bar"
+    output: ['baz']
+    shell:
+        '''
+        export DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS=foo,bar
+        export DUMMY_CONTAINER_OUTPUT_PATHS=baz
+        export DUMMY_CONTAINER_DIAGNOSTICS_DIRECTORY=spam
+        export eggs=coconut
+        echo hello world
```

Comment:
needs a closing ''' I think? How is this passing?

Reply:
Ugh, it's because I'm only looping over the lines in the expected output. I added a check that they have the same number of lines